pkg/osgi/config.go

```diff
@@ -43,6 +46,15 @@ func (c ConfigListItem) PropertyValues() map[string]any {
 	return result
 }
 
+func (c ConfigListItem) AemcId() string {
```

Comment:
it's more like returning `FactoryPid()` isn't it? `aemcId` prop is used internally

how about `strcase.ToCamel(common.AppId) + "~"`